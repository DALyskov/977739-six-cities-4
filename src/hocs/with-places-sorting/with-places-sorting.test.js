import React from 'react';
import renderer from 'react-test-renderer';

import withPlacesSorting from './with-places-sorting.jsx';

const MockComponent = () => {
  return <div></div>;
};

const MockComponentWrapped = withPlacesSorting(MockComponent);

describe(`WithPlacesSorting_snapchots`, () => {
  it(`WithPlacesSorting_is_rendered_correctly`, () => {
    const tree = renderer.create(<MockComponentWrapped />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
