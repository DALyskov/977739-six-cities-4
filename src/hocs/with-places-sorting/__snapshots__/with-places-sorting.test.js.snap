// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithPlacesSorting_snapchots WithPlacesSorting_is_rendered_correctly 1`] = `<div />`;
